deploy/monitor/Dockerfile

```diff
@@ -2,39 +2,12 @@ FROM node:21-bookworm
 
```

Comment:
Since this is so trimmed down now I wonder if we should just use the Playwright-provided image e.g. `FROM mcr.microsoft.com/playwright:v???-jammy`, I imagine then we wouldn't need to install the browsers and such

Reply:
Ha, that's a good idea. I originally used that and then switched to be able to install the UF and AWS CLI. We can definitely switch back.

Reply:
Oh, I do remember one reason I wanted to move off of the official image is that we only need one browser. Their official image is just node bookworm with all of the browsers.